src/aat/resources/features/F-049/S-232.td.json

```diff
@@ -0,0 +1,23 @@
+{
+  "_guid_": "S-232",
+  "_extends_": "F-049_Test_Data_Base",
+  "title": "must return 403 when request provide unauthorized access",
+  "specs": [
+    "a detailed profile in CCD",
+    "Start case creation as Case worker",
+    "CCD Data Store",
```

Comment:
What are these?

Reply:
as the above is an array we cant move some part to base data, framework dependency

Reply:
that wasn't the point